src/DashboardService.Models.Dto/Requests/Comment/PatchCommentRequest.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace LT.DigitalOffice.DashboardService.Models.Dto.Requests.Comment;
+
+public record PatchCommentRequest
+{
+  public Guid TaskId { get; set; }
```

Comment:
Have you ever seen a feature for transfering your comment to another post/smth???

Reply:
++